x-pack/plugins/infra/common/formatters/index.ts

```diff
@@ -34,5 +33,5 @@ export const createFormatter = (format: InventoryFormatterType, template: string
   }
   const fmtFn = FORMATTERS[format];
   const value = fmtFn(Number(val));
-  return Mustache.render(template, { value });
+  return template.replace(/{{value}}/g, value);
```

Comment:
:thought_balloon: Too bad `replaceAll()` is not yet available in our node version. It would allow us to avoid constructing a regex.

Reply:
yep :( I initially used that but the CI crashed